DirectProgramming/C++SYCL_FPGA/Tutorials/Features/max_interleaving/CMakeLists.txt

```diff
@@ -113,7 +113,7 @@ else()
     set(SEED_FLAG "-Xsseed=${SEED}")
 endif()
 
-set(COMMON_COMPILE_FLAGS -fsycl -fintelfpga -Wall ${WIN_FLAG} ${DEBUG_FLAGS} ${QACTYPES} ${USER_FLAGS})
+set(COMMON_COMPILE_FLAGS -fsycl -fintelfpga -Wall -Wno-unused-label ${WIN_FLAG} ${DEBUG_FLAGS} ${QACTYPES} ${USER_FLAGS})
```

Comment:
Please respect the 'No changes after here' comment; this part of the CMake file ought to be consistent across all code samples, so any changes to it should be universally beneficial. If you make changes to this section of the CMake file, then you should upstream them to the `fpga_template` code sample as well so that future code samples get this benefit. 

If you think this flag only applies to this code sample, please add it to line 42 instead like this:

```
set(USER_FLAGS -Wno-unused-label;${USER_FLAGS})
```

Personally, I feel like if we are going to add `-Wno-unused-label` to all samples, we would do better to simply remove `-Wall` from the template.

Reply:
Thanks Paul - It's specific to this one sample, as this one uses a few loop labels in the source code. I have updated the CMakefiles.